### PR TITLE
update README.md and add setup scripts

### PR DESCRIPTION
**Pull request by Fijo-Lurk**
setup direction and update package.json with new script to start and install project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +12,27 @@ The goal of the project is to create a complete application with a limited numbe
...
+
+# create development env and start in watch mode
+$ npm run env:development
+$ NODE_ENV=development npm run start:server
+
+# create prod env and start in production mode
+$ npm run env:production
+$ NODE_ENV=production npm run start:prod
+
+$ npm run build
```
Comment (mblomdahl) [inline README.md:33]: This should probably be moved to the beginning of the "_Running the App_" block, else we will start up a NestJS server with no Angular build available. Or, if you prefer, maybe replace "_Installation_" block with a 

```md
## Setup

```bash
$ npm run install && npm run build
...
```

File: package.json [hunk 2]
```diff
@@ -4,7 +4,12 @@
     "start:frontend": "cd frontend && ng serve",
     "build:frontend": "cd frontend && npm run build",
     "build:server": "cd server && npm run build",
-    "build": "npm run build:server && npm run build:frontend"
+    "build": "npm run build:server && npm run build:frontend",
+    "install:ui": "cd frontend && npm install",
```
Comment (mblomdahl) [inline package.json:8]: I'd propose to be consistent about calling it "`ui`", not a mix of sometimes "frontend" sometimes "ui". Same for the root folder, `frontend/` → `ui/`!

File: Server/config/env/template.env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+
+WEB_CLIENT_HOST=YOUR_CLIENT_URL
```
Comment (mblomdahl) [inline Server/config/env/template.env:3]: Notice the naggy red circle at the end of the line here. It indicates one should make sure to have `insert_final_newline = true` configured in the EditorConfig!

It is a bit hard to understand what goes in here also, without digging into the server implementation details. Maybe `http://localhost:4200` would be a sensible default?

**Review by mblomdahl — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Fijo-Lurk) [reply package.json:8]: changing it to ui to be more consistent

**Pull request merged**